docs/software/distributed_projects.md

```diff
@@ -0,0 +1,196 @@
+
+=== "Logging"
+
+    Folding@Home has a logging feature which outputs into the journal logging system. The logs can be viewed via:
+
+    ```sh
+    journalctl -u fahclient
+    ```
+
+=== "Influence the computational power"
+
+    In addition to the *power slider* in the web interface the user can influence the number of jobs that run on the system.  
+    The example config given below has 3 job slots, ideal for a 6 core CPU (allowing 2 CPU cores per job slot):
+
+    ```xml
+    <config>
+      <!-- Slot Control -->
+      <power v='FULL'/>
+
+      <!-- User Information -->
+      <user v='Fourdee'/>
+
+      <!-- Folding Slots -->
+      <slot id='0' type='CPU'/>
+      <slot id='1' type='CPU'/>
+      <slot id='2' type='CPU'/>
+    </config>
+    ```
+
+## Tor Relay
+
+![Tor logo](../assets/images/dietpi-software-tor-logo.png){: width="150" height="91" loading="lazy"}
+
+Contribute a node to the Tor network, which allows people to be anonymous on the internet.
+
+=== "Types of relay"
+
+    You can run many types of relay, each with their own technical requirements and legal implications.
+
+    Bridges are the safest relay to run from home, and are relatively easy, low-risk and low bandwidth, but they have a big impact on users, especially in censored countries. Normally, IP addresses and other information of Tor relays is published, making it easy for websites to blacklist the relay, and anything else using that IP address. Since a bridge isn't listed publicly, it is unlikely to be blocked by websites or receive abuse complaints.
+
+    Guard/Middle relays are the first and second relays connected to, respectively. Information about them is listed, but they are unlikely to receive abuse complaints. However, they may be blocked by certain services that don't understand how Tor works or deliberately want to censor Tor users. If you have one static IP address, consider running a bridge instead.
+
+    Exit relays are the final relay connected to, and the one that actually sends traffic to its destination. The website will see the exit relay's IP address instead of the real IP address of the Tor user. Exit relays have the greatest legal exposure and liability of all the relays, and should not be run from home.
+
+    ??? info "Exit relay prep"
+        Running exit relays requires some preparation.
+        Before running an exit relay, you should set a reverse DNS (and, if possible, WHOIS) record to make it clearer that your IP address is a Tor exit relay.
+        Also, it is recommended that you run an exit relay on its own server, with its own IP address.
+
+=== "Monitoring"
+
+    The simplest way to monitor the Tor relay is to use the DietPi-CloudShell scene, for this run `dietpi-cloudshell` from command line and select the "Tor Relay" scene. It uses the Tor monitor [nyx](https://nyx.torproject.org/) which can be called in standalone mode by executing `nyx` from command line.
```

Comment:
Harmonization:

> ... for this run 
> 
> ```
> dietpi-cloudshell
> ``` 
> 
> from the command line ...
>

Reply:
I think for such single-word commands in the middle of a sentence, inline code is okay.